contents/newsletter/remote-working.md

```diff
@@ -0,0 +1,85 @@
+title: The best habits of the best remote teams
```

Comment:
```suggestion
title: The habits if effective remote teams
```

Had best twice

Reply:
I still think 'The habits of effective remote teams' lacks a certain something but I am struggling to think of an alternative.

How the best/most effective remote teams operate
Lessons from the most effective remote teams
What you can learn from the most effective remote teams

Reply:
How the best remote teams actually work (/operate)

Reply:
quite like

How the best remote teams actually work

don't feel 'habits' is a good word for display copy

Reply:
I've put them both in the poll. Can pick the winner.